docs/src/user-docs/resources-datasets.md

```
@@ -32,3 +33,5 @@ For evaluation results comparing CLP and other tools, see our
 [6]: https://zenodo.org/records/10516226
 
 [7]: https://zenodo.org/records/10516345
+
+[8]: https://zenodo.org/records/10516401
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)